util/db.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 const dbPath = "./db"
-const defaultUsername = "admin"
-const defaultPassword = "admin"
+const defaultUsername = os.Getenv("USER")
+const defaultPassword = os.Getenv("PASSS")
```

Comment:
What if we run the binary file directly on the host instead of using Docker? Should we store the default username & password in the code?

Reply:
It would probably make sense to change it so the environment variable names are more unique than USER and PASS, and then set the default values from code if the env values did not exist.

That would mean that we don't modify the ```Dockerfile``` since the default is in code.